UnitySDK/Assets/ML-Agents/Examples/Tennis/Scripts/HitWall.cs

```diff
@@ -4,122 +4,125 @@ public class HitWall : MonoBehaviour
 {
     public GameObject areaObject;
     public int lastAgentHit;
+    public int lastFloorHit;
```

Comment:
Could this be made into an enum to make it more clear what these numbers mean?
Perhaps something like this:
```suggestion
    public enum FloorHit
        {
            FloorAHit,
            FloorBHit,
            FloorHitUnset
        }
        
    public FloorHit lastFloorHit;
```